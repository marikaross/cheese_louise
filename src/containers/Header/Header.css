.header {
  width: 100vw;
  height: 7%;
  background-image: url('../../images/France_1024x1024.jpg');
  background-repeat: no-repeat; 
  background-position: center;
  background-size: cover;
}


.header-link {
  text-decoration: none;
  font-family: 'Rokkit', serif;
  font-size: 3em;
  padding: 20px;
  color: #811b13;
}

.favorites-link {
  text-decoration: none;
  float: right;

}