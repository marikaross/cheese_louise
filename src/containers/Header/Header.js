import React from 'react';
import { connect } from 'react-redux';
import { NavLink } from 'react-router-dom';
import PropTypes from 'prop-types';
import './Header.css';

export const Header = () => {

  return(
      <div className='header'>
        <div className='header-link'>
          <NavLink className='logo'to='/'>Cheese-Louise</NavLink>
        </div>
        <div className='faves'>
          <NavLink className='favorites-link' to='/favorites'>Favorites</NavLink>
        </div >
      </div>
    )
}