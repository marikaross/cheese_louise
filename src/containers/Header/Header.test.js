import { Header } from './Header';
import React from 'react';
import { shallow } from 'enzyme';

describe('Header', () => {

  it('should match the snapShot', () => {
    let wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})