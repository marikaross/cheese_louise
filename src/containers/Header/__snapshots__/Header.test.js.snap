// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapShot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="header-link"
>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    className="logo"
                    to="/"
          >
                    Cheese-Louise
          </NavLink>
</div>,
        <div
          className="faves"
>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    className="favorites-link"
                    to="/favorites"
          >
                    Favorites
          </NavLink>
</div>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            aria-current="page"
            className="logo"
            to="/"
>
            Cheese-Louise
</NavLink>,
          "className": "header-link",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Cheese-Louise",
            "className": "logo",
            "to": "/",
          },
          "ref": null,
          "rendered": "Cheese-Louise",
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            aria-current="page"
            className="favorites-link"
            to="/favorites"
>
            Favorites
</NavLink>,
          "className": "faves",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "aria-current": "page",
            "children": "Favorites",
            "className": "favorites-link",
            "to": "/favorites",
          },
          "ref": null,
          "rendered": "Favorites",
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="header-link"
>
            <NavLink
                        activeClassName="active"
                        aria-current="page"
                        className="logo"
                        to="/"
            >
                        Cheese-Louise
            </NavLink>
</div>,
          <div
            className="faves"
>
            <NavLink
                        activeClassName="active"
                        aria-current="page"
                        className="favorites-link"
                        to="/favorites"
            >
                        Favorites
            </NavLink>
</div>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              aria-current="page"
              className="logo"
              to="/"
>
              Cheese-Louise
</NavLink>,
            "className": "header-link",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "Cheese-Louise",
              "className": "logo",
              "to": "/",
            },
            "ref": null,
            "rendered": "Cheese-Louise",
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              aria-current="page"
              className="favorites-link"
              to="/favorites"
>
              Favorites
</NavLink>,
            "className": "faves",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": "Favorites",
              "className": "favorites-link",
              "to": "/favorites",
            },
            "ref": null,
            "rendered": "Favorites",
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
